src/adera/Machines/Rocket.cpp

```diff
@@ -222,12 +223,12 @@ void SysMachineRocket::attach_plume_effect(ActiveEnt ent)
 
     if (plumeNode == entt::null)
     {
-        std::cout << "ERROR: could not find plume anchor for MachineRocket "
-            << ent << "\n";
+        m_scene.get_application().get_logger()->error("ERROR: could not find plume anchor for MachineRocket {}", ent);
         return;
     }
-    std::cout << "MachineRocket " << ent << "'s associated plume: "
-        << plumeNode << "\n";
+
+    m_scene.get_application().get_logger()->info("MachineRocket {0}\'s associated plume: {1}", ent, plumeNode);
```

Comment:
why are you doing {0} and {1} here but in other places you don't do that, and just use {}.